### PR TITLE
#4 views

### PR DESCRIPTION
**Pull request by JarodXP**
@Wyona: please check the banch, thanks

### CODE REVIEW

File: src/App/Admin/ProfessionalController.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        {
+            echo $this->twigEnvironment->render('/professional.html.twig');
+        }
+        catch (LoaderError | RuntimeError | SyntaxError $e)
+        {
+            print_r($e->getMessage());
+        }
+    }
+
+    public function registerProfessionalAction()
```
Comment (Tutt-J) [inline src/App/Admin/ProfessionalController.php]: What's this ?

File: src/App/Admin/ProfessionalController.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<?php
```
Comment (Tutt-J) [inline src/App/Admin/ProfessionalController.php]: What's this controller ?

File: src/App/Admin/ProfessionalController.php [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        {
+            echo $this->twigEnvironment->render('/social.html.twig');
+        }
+        catch (LoaderError | RuntimeError | SyntaxError $e)
+        {
+            print_r($e->getMessage());
+        }
+    }
+
+    public function addNetworksAction()
```
Comment (Tutt-J) [inline src/App/Admin/ProfessionalController.php]: What's this ?

File: src/App/Admin/ProfessionalController.php [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    {
+
+    }
+
+    public function removeNetworksAction()
```
Comment (Tutt-J) [inline src/App/Admin/ProfessionalController.php]: What's this ?

File: src/config/routes.yml [hunk 5]
```diff
@@ -68,6 +68,28 @@
   controller: profile
   action: editProfile
 
+  #PROFESSIONAL
```
Comment (Tutt-J) [inline src/config/routes.yml]: What do you want to say by "Profesional" ?

File: src/App/Shared/Views/baseLayout.html.twig [hunk 6]
```diff
@@ -40,7 +37,7 @@
 
 {% block footer %}{% endblock %}
 
-{% block footerScripts %}
+{% block scripts %}
```
Comment (Tutt-J) [inline src/App/Shared/Views/baseLayout.html.twig]: Why not user footerScripts ??

File: src/App/Admin/ProfessionalController.php (hunk 1 above)
Comment (JarodXP) [reply src/App/Admin/ProfessionalController.php]: I've created a new tab in the admin page called "professional".
It's the place where I manage social networks and the resume.

File: src/App/Admin/ProfessionalController.php (hunk 2 above)
Comment (JarodXP) [reply src/App/Admin/ProfessionalController.php]: see above

File: src/App/Admin/ProfessionalController.php (hunk 3 above)
Comment (JarodXP) [reply src/App/Admin/ProfessionalController.php]: In the professional tab, it's possible to add a network with name and icon to be displayed in the frontoffice

File: src/App/Admin/ProfessionalController.php (hunk 4 above)
Comment (JarodXP) [reply src/App/Admin/ProfessionalController.php]: see above

File: src/config/routes.yml (hunk 5 above)
Comment (JarodXP) [reply src/config/routes.yml]: see above

File: src/App/Shared/Views/baseLayout.html.twig (hunk 6 above)
Comment (JarodXP) [reply src/App/Shared/Views/baseLayout.html.twig]: First i thought it was a "found and replace" mistake from my part, so I corrected it.
When I saw it was a change from you I forgot to change it back.

Comment (Tutt-J) [reply src/App/Shared/Views/baseLayout.html.twig]: Ok so change it please

**Comment by JarodXP** [comment]
@Tutt-J : Hi, I've passed W3C validations (both HTML and CSS) and made the requested changes.
I've also added some JS for the sidebar in frontoffice.

**Comment by JarodXP** [comment]
No i didn't forget, I just didn't put a meta in the admin pages as it's
only for seo

**Comment by JarodXP** [comment]
> 
> 
> Cache directory missing....
_Empty cache file added to register cache directory in the commit (empty directories are not registered)_ 

> For template please have a coherent naming because sometimes you say adminEditPost and the nyou say newPost.html. if you want that all your admin template begin by admin, it will be all the same.
_ok, done_

> Do you change $_ENV (charset and local) to $GLOBALS ?
_No I let it as you did, as we discussed._

> Have you change database model ?
_yes, but it's not part of that PR_

**Comment by Tutt-J** [comment]
> 
> 
> > Cache directory missing....
> > _Empty cache file added to register cache directory in the commit (empty directories are not registered)_
> _**I will create an issue for this and explain you later.**_
> > Do you change $_ENV (charset and local) to $GLOBALS ?
> > _No I let it as you did, as we discussed._
>  _**Yes we discuss and i say you to do this as you propose, so I don't understand**_
> > Have you change database model ?
> > _yes, but it's not part of that PR_
 _**Yes sure but can you send me by email ?**_

**Comment by JarodXP** [comment]
@Tutt-J : Ok, $_ENV changed into $GLOBALS

**Pull request merged**